### ISSUE TITLE
Does not render dict fields

### ISSUE THREAD

**Issue by mekhlakapoor** [AllenNeuralDynamics/aind-metadata-entry-js#91]
**Describe the bug**
In the ophys rig schema, the field "software parameters" expects a dictionary. However, the app renders no input field for it. Because there is no input area for this field, users are unable to submit ophys rig forms

**Expected behavior**
Find out what the expected dict looks like

**Screenshots**
![Screen Shot 2023-06-29 at 11 26 18 AM](https://github.com/AllenNeuralDynamics/aind-metadata-entry-js/assets/54870020/55231b7e-c4c5-4f4b-b50c-6b295130391a)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by jtyoung84** [comment]
This might be difficult to do in the ui. You can add a property to the schema to let the ui know which type of object to render:
```
class Software(AindModel):
    """Description of generic software"""

    name: str = Field(..., title="Software name")
    version: str = Field(..., title="Software version")
    parameters: Optional[dict] = Field(..., title="Software parameters", additionalProperties={"type": "string"})
```

**Comment by jtyoung84** [comment]
@mekhlakapoor I created this ticket if you want to update the schema while fixing some other issues with `device.py`:
https://github.com/AllenNeuralDynamics/aind-data-schema/issues/390

**Comment by mekhlakapoor** [comment]
Closed because its issue #390 on aind-data-schema
